GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -995,7 +995,7 @@ public void loadTexture(TextureCallback callback,
      * {@link #loadTexture(GVRAndroidResource.TextureCallback, GVRAndroidResource, int)
      * : you don't have to implement a callback; you don't have to pay attention
      * to the low-level details of
-     * {@linkplain GVRSceneObject#attachRenderData(GVRRenderData) attaching} a
+     *{@linkplain GVRSceneObject#attachRenderData(GVRRenderData) attaching} a
```

Comment:
This is generated by Ctrl+Shift+F. Even if I manually add the blank back, it will be removed by Ctrl+Shift+F. :(